ddtrace/contrib/unittest/patch.py

```diff
@@ -230,6 +230,45 @@ def _is_invoked_by_cli(instance: unittest.TextTestRunner) -> bool:
     )
 
 
+def _extract_test_method_object(test_object):
+    test_method_object = None
+    if hasattr(test_object, "_testMethodName"):
+        test_method_object = getattr(test_object, test_object._testMethodName, None)
+    return test_method_object
+
+
+def _get_source_file_path_for_test_method(test_object):
+    test_name = _extract_test_method_name(test_object)
+    test_method_object = _extract_test_method_object(test_object)
+    source_file_path = os.path.relpath(inspect.getfile(test_method_object))
+    if not test_method_object:
```

Comment:
I'd move this above, since you're using `test_method_object` before checking if it's not truthy.